.github/workflows/download_upload_cycle.yml

```diff
@@ -0,0 +1,45 @@
+name: download_upload_cycle
+on:
+  pull_request:
+  push:
+  schedule:
+    # once per hour
+    - cron: "0 * * * *"
```

Comment:
We currently have 30ish samples on the public  instance, and we've already run this a bunch.  I think we can start with this on a daily cycle.  Or maybe just have a follow on PR with the change so we have a few more runs before it is reduced to daily.

Reply:
Ok, changed to once per day in [ea065bb](https://github.com/girder/shapeworks-cloud/pull/291/commits/ea065bba3cf49eb3980761f13dee90a9fe55f350)